.claude/hooks/SessionStart/invoke_context_loader.py

```
@@ -0,0 +1,136 @@
+
+
+def main() -> int:
+    """Load HANDOFF.md and latest retro, print to stdout for context injection."""
+    # Skip if stdin is a TTY (interactive, not piped from Claude)
+    if sys.stdin.isatty():
+        return 0
+
+    # Read stdin (Claude provides JSON context)
+    try:
+        stdin_data = sys.stdin.read()
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)